docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -380,10 +447,23 @@ When you run the `create_service_account` module as part of the playbook at the
 ----
 . In the `openshift_offline_token` field, enter your OpenShift offline token.
 . To directly specify a Kafka instance ID, enter a value in the `kafka_id` field. Otherwise, Ansible gets the Kafka ID from the `kafka_req_resp.id` variable.
```

Comment:
Is it worth pulling the topic deletion steps out into their own procedure, like you did for the service account and Kafka instance? It feels a _wee_ bit odd to have steps for creating, modifying, and then deleting a topic within the same procedure, particularly given that the abstract talks only about creating one.

Reply:
That was why I had the title as 'Configuring' but I can break the deletion part out. Was concerned it was becoming too repetitive this way.